x-pack/plugins/security_solution/public/timelines/components/side_panel/event_details/expandable_event.tsx

```diff
@@ -63,11 +59,15 @@ const StyledEuiFlexItem = styled(EuiFlexItem)`
 `;
 
 export const ExpandableEventTitle = React.memo<ExpandableEventTitleProps>(
-  ({ isAlert, loading, handleOnEventClosed }) => (
+  ({ isAlert, loading, handleOnEventClosed, ruleName }) => (
     <StyledEuiFlexGroup gutterSize="none" justifyContent="spaceBetween" wrap={true}>
       <EuiFlexItem grow={false}>
         <EuiTitle size="s">
-          {!loading ? <h4>{isAlert ? i18n.ALERT_DETAILS : i18n.EVENT_DETAILS}</h4> : <></>}
+          {!loading ? (
+            <h4>{isAlert && !isEmpty(ruleName) ? ruleName : i18n.EVENT_DETAILS}</h4>
+          ) : (
+            <></>
```

Comment:
Do you know why this is needed here? Could we just do
```
{!loading && (
	<h4>{isAlert && !isEmpty(ruleName) ? ruleName : i18n.EVENT_DETAILS}</h4>
)}
```

Reply:
Yeah, EuiTitle needs to have a child in it anyway. My intention was use the empty span as a place holder to avoid the title height jumping on loaded, but seems that It works the same as 
```
{!loading && (
          <EuiTitle size="s">
            <h4>{isAlert && !isEmpty(ruleName) ? ruleName : i18n.EVENT_DETAILS}</h4>
          </EuiTitle>
        )}
```
so I'll just remove the place holder and hide the entire title.